### PR TITLE
installing: OKD: Add note about Ignition SSH keys on FCOS Follow Up

### PR DESCRIPTION
**Pull request by mburke5678**
Editing text added to docs in https://github.com/openshift/openshift-docs/pull/28387# and moved per @codyhoag

### CODE REVIEW

**Comment by mburke5678** [comment]
@codyhoag Would you mind reviewing?

File: modules/ssh-agent-using.adoc [hunk 1]
```diff
@@ -102,6 +102,14 @@ link:https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/ec2-key-pairs.html[AWS
 ====
 endif::[]
 
+ifdef::openshift-origin[]
+[NOTE]
+====
+On clusters running {op-system-first}, the SSH keys specified in the Ignition config files are written to the `/home/core/.ssh/authorized_keys.d/core` file. However, the Machine Config Operator manages SSH keys in `/home/core/.ssh/authorized_keys` file and configures *sshd* to ignore the `/home/core/.ssh/authorized_keys.d/core` file.
```
Comment (codyhoag) [inline modules/ssh-agent-using.adoc:108]:
```suggestion
On clusters running {op-system-first}, the SSH keys specified in the Ignition config files are written to the `/home/core/.ssh/authorized_keys.d/core` file. However, the Machine Config Operator manages SSH keys in the `/home/core/.ssh/authorized_keys` file and configures *sshd* to ignore the `/home/core/.ssh/authorized_keys.d/core` file.
```

**Review by codyhoag — COMMENTED** [review]
One nit; looks good otherwise!

**Pull request merged**